Unused variables in support_delaunay_shewchuk

Set LICENSE_RESTRICTED to GPL, then you'll see following warnings:
```
../../src/gmt_support.c:3798:77: warning: unused parameter 'x_in' [-Wunused-parameter]
GMT_LOCAL uint64_t support_delaunay_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, int **link) {
                                                                            ^
../../src/gmt_support.c:3798:91: warning: unused parameter 'y_in' [-Wunused-parameter]
GMT_LOCAL uint64_t support_delaunay_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, int **link) {
                                                                                          ^
../../src/gmt_support.c:3798:106: warning: unused parameter 'n' [-Wunused-parameter]
GMT_LOCAL uint64_t support_delaunay_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, int **link) {
                                                                                                         ^
../../src/gmt_support.c:3798:115: warning: unused parameter 'link' [-Wunused-parameter]
GMT_LOCAL uint64_t support_delaunay_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, int **link) {
                                                                                                                  ^
../../src/gmt_support.c:3804:88: warning: unused parameter 'x_in' [-Wunused-parameter]
GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
                                                                                       ^
../../src/gmt_support.c:3804:102: warning: unused parameter 'y_in' [-Wunused-parameter]
GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
                                                                                                     ^
../../src/gmt_support.c:3804:117: warning: unused parameter 'n' [-Wunused-parameter]
GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
                                                                                                                    ^
../../src/gmt_support.c:3804:128: warning: unused parameter 'wesn' [-Wunused-parameter]
GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
                                                                                                                               ^
../../src/gmt_support.c:3804:147: warning: unused parameter 'mode' [-Wunused-parameter]
GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
                                                                                                                                                  ^
9 warnings generated.
```

--- a/src/gmt_support.c
+++ b/src/gmt_support.c
@@ -3796,12 +3796,21 @@ GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, d
 #else
 /*! Dummy functions since not installed */
 GMT_LOCAL uint64_t support_delaunay_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, int **link) {
+	gmt_M_unused (x_in);
+	gmt_M_unused (y_in);
+	gmt_M_unused (n);
+	gmt_M_unused (link);
 	GMT_Report (GMT->parent, GMT_MSG_ERROR, "unavailable: Shewchuk's triangle option was not selected during GMT installation\n");
 	return (0);
 }
 
 /*! . */
 GMT_LOCAL struct GMT_DATASET * support_voronoi_shewchuk (struct GMT_CTRL *GMT, double *x_in, double *y_in, uint64_t n, double *wesn, unsigned int mode) {
+	gmt_M_unused (x_in);
+	gmt_M_unused (y_in);
+	gmt_M_unused (n);
+	gmt_M_unused (wesn);
+	gmt_M_unused (mode);
 	GMT_Report (GMT->parent, GMT_MSG_ERROR, "unavailable: Shewchuk's triangle option was not selected during GMT installation\n");
 	return (NULL);
 }

